### PR TITLE
Lower logging level of some common logs.

### PR DESCRIPTION
**Pull request by vmarmol**
/cc @rjnagal

### CODE REVIEW

File: manager/container.go [hunk 1]
```diff
@@ -320,7 +320,7 @@ func (c *containerData) updateStats() error {
 		err := c.summaryReader.AddSample(*stats)
 		if err != nil {
 			// Ignore summary errors for now.
-			glog.V(2).Infof("failed to add summary stats for %q: %v", c.info.Name, err)
+			glog.V(4).Infof("failed to add summary stats for %q: %v", c.info.Name, err)
```
Comment (rjnagal) [inline manager/container.go]: Is this common? Is it for dead containers?

Comment (vmarmol) [inline manager/container.go]: It is not, will remove.

**Comment by vmarmol** [comment]
Ping @rjnagal 

e2e's are failing due to an unrelated issue. Fixing in another PR.

**Comment by rjnagal** [comment]
LGTM

**Pull request merged**